frontend/src/components/mine/Tailings/MineTailingsInfo.js

```diff
@@ -143,6 +145,20 @@ export class MineTailingsInfo extends Component {
     }
   }
 
+  removeReport = (event, exp_doc_guid) => {
+    event.preventDefault();
+    this.props.removeExpectedDocument(exp_doc_guid).then(() => {
+      this.props.fetchMineRecordById(this.props.mine.guid);
+    });
+  };
+
+  getFileFromDocumentManager(doc_mgr_guid) {
```

Comment:
Unless this param needs to be in snake_case, (ie you are getting it from the backend, or using it in an api endpoint) try to name everything using camelCase,